e2e/deno-e2e/tests/deno.spec.ts

```diff
@@ -38,26 +38,23 @@ describe('Deno', () => {
 
   describe('application', () => {
     it('should create deno app', async () => {
-      await runNxCommandAsync(`generate @nrwl/deno:app ${appName}`);
+      // await runNxCommandAsync(`generate @nrwl/deno:app ${appName}`);
+      await runNxCommandAsync(`generate @nrwl/deno:preset ${appName}`);
       expect(readJson(`import_map.json`)).toEqual({ imports: {} });
-      expect(readJson(`apps/${appName}/deno.json`)).toEqual({
-        importMap: '../../import_map.json',
+      expect(readJson(`deno.json`)).toEqual({
+        importMap: 'import_map.json',
```

Comment:
can we have separate tests for standalone deno and integrated, so we are covered in both cases instead of just the standalone?